### ISSUE TITLE
Calibur theme not showing Discover (Random Books)

### ISSUE THREAD

**Issue by aCID-sLAM** [janeczku/calibre-web#1741]
**To Reproduce**
1. Switch to dark calibur
2. No Random Books
3. Switch to classic theme
4. Random books

**Logfile**
No entry to theme

**Expected behavior**
Random Books show at first line

**Environment (please complete the following information):**
 - OS: Windows

**Comment by OzzieIsaacs** [comment]
For me it's working:
![grafik](https://user-images.githubusercontent.com/6438895/101154273-cb740400-3625-11eb-8499-1311071002dd.png)

If you are referring to the books in overview:
![grafik](https://user-images.githubusercontent.com/6438895/101154340-e8103c00-3625-11eb-89ba-5dfdd6ff5721.png)
This is intentionally (I think)

**Comment by aCID-sLAM** [comment]
Yeah, the overview is what i ment, i wasnt precise enough. Shouldnt the overview be the same, on all themes?
Atleast i would expect that, as a user. As now, switching the theme, controls the first login experience of the user, regarding the Random Books.
